sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -544,12 +493,61 @@ public static JsonSerializable instantiateJsonSerializable(ObjectNode objectNode
                 || klassType.equals(DatabaseAccount.class) || klassType.equals(DatabaseAccountLocation.class)
                 || klassType.equals(ReplicationPolicy.class) || klassType.equals(ConsistencyPolicy.class)
                 || klassType.equals(DocumentCollection.class) || klassType.equals(Database.class)) {
-                return (JsonSerializable) klassType.getDeclaredConstructor(ObjectNode.class).newInstance(objectNode);
+                Constructor declaredConstructor =
+                    klassType.getDeclaredConstructor(ObjectNode.class);
+                declaredConstructor.setAccessible(true);
```

Comment:
can you move `instantiateJsonSerializable` to model package?
If you do that then you don't have to do `setAccessible()`

Reply:
Agreed, will move it.